WcaOnRails/spec/controllers/api_controller_spec.rb

```diff
@@ -158,6 +158,14 @@ def api_sign_in_as(user, scopes: nil)
       expect(json["result"].count { |r| r["class"] == "user" }).to eq 0
       expect(json["result"].count { |r| r["class"] == "person" }).to eq 1
     end
+
+    it "works well when parts of the name are given" do
```

Comment:
With only one user in the DB, this test would pass with the implementation `Person.all`.

I'd add 1-2 more people, and test more cases that demonstrate what the search does and doesn't do.